src/olympia/abuse/tests/test_serializers.py

```diff
@@ -16,8 +16,45 @@
 
 
 class TestAddonAbuseReportSerializer(TestCase):
-    def serialize(self, report, **extra_context):
-        return AddonAbuseReportSerializer(report, context=extra_context).data
+    def serialize(self, report, context=None):
+        return dict(AddonAbuseReportSerializer(report, context=context or {}).data)
```

Comment:
Why the wrapping in `dict`?  I've not seen this pattern elsewhere in our tests.

Reply:
Convenience - `<serializer.data>` is an `OrderedDict` and so if the `assert` fails, it doesn't have as nice diffs as if it was a `dict` comparison - it shows the diff as a tuple of tuples instead IIRC. I had an issue with the patch initially and that helped narrow it down.